src/bin/cql-stress-cassandra-stress/settings/option/mode.rs

```diff
@@ -0,0 +1,212 @@
+use anyhow::{Context, Result};
+use scylla::{frame::Compression, transport::session::PoolSize};
+
+use crate::settings::{
+    param::{
+        types::{ConnectionsPerHost, ConnectionsPerShard},
+        ParamsParser, SimpleParamHandle,
+    },
+    ParsePayload,
+};
+
+pub struct ModeOption {
+    pub compression: Option<Compression>,
+    pub user_credentials: Option<(String, String)>,
+    pub pool_size: PoolSize,
+}
```

Comment:
Instead of using an unnamed tuple for credentials and relying on implicit ordering of it, I suggest creating a dedicated struct for them here.

Reply:
```suggestion

struct Credentials {
   username: String,
   password: String,
}

pub struct ModeOption {
    pub compression: Option<Compression>,
    pub user_credentials: Option<Credentials>,
    pub pool_size: PoolSize,
}
```